include/picongpu/fields/incidentField/profiles/ExpRampWithPrepulse.hpp

```diff
@@ -71,6 +71,12 @@ namespace picongpu::fields::incidentField
                 static constexpr float_X endUpramp = TIME_PEAKPULSE - 0.5_X * LASER_NOFOCUS_CONSTANT;
                 static constexpr float_X startDownramp = TIME_PEAKPULSE + 0.5_X * LASER_NOFOCUS_CONSTANT;
 
+                /** Pulse duration
+                         *

```

Comment:
the same her 
(And if I check your code I get some strange carrier return symbols - not sure if they come from my or your side)